packages/thirdweb/src/wallets/in-app/core/authentication/index.ts

```diff
@@ -1,32 +1,41 @@
 import type { ThirdwebClient } from "../../../../client/client.js";
+import { isBrowser } from "../../../../utils/platform.js";
+import type { InAppConnector } from "../interfaces/connector.js";
 import {
+  type AuthArgsType,
   type AuthLoginReturnType,
   AuthProvider,
-} from "../../implementations/interfaces/auth.js";
-import { UserWalletStatus } from "../../implementations/interfaces/in-app-wallets/in-app-wallets.js";
-import {
-  loginWithPasskey,
-  registerPasskey,
-} from "../../implementations/lib/auth/passkeys.js";
-import type { InAppWalletSdk } from "../../implementations/lib/in-app-wallet.js";
-import type { AuthArgsType, PreAuthArgsType } from "./type.js";
+  type PreAuthArgsType,
+  UserWalletStatus,
+} from "./type.js";
 
-const ewsSDKCache = new WeakMap<ThirdwebClient, InAppWalletSdk>();
+const ewsSDKCache = new WeakMap<ThirdwebClient, InAppConnector>();
 
 /**
  * @internal
  */
-async function getInAppWalletSDK(client: ThirdwebClient) {
+async function getInAppWalletConnector(client: ThirdwebClient) {
   if (ewsSDKCache.has(client)) {
-    return ewsSDKCache.get(client) as InAppWalletSdk;
+    return ewsSDKCache.get(client) as InAppConnector;
+  }
+
+  let ewSDK: InAppConnector;
+  if (isBrowser()) {
+    const { InAppWebConnector } = await import(
+      "../../web/lib/web-connector.js"
+    );
+    ewSDK = new InAppWebConnector({
+      client: client,
+    });
+  } else {
+    const { InAppNativeConnector } = await import(
```

Comment:
this means if someone (for some reason) calls this in node we end up in the native path, likely we want to explicitly handle both web and native and then throw an error in node (for now) - we actually *can* make this work in node though right?

Reply:
node not supported right now, will throw